bytecomp/translmod.ml

```diff
@@ -419,26 +426,27 @@ and transl_structure loc fields cc rootpath = function
           let v = Array.of_list (List.rev fields) in
           let get_field pos = Lvar v.(pos)
           and ids = List.fold_right IdentSet.add fields IdentSet.empty in
-          let (result, names) = List.fold_right
-              (fun  (pos, cc) (code, name) ->
+          let get_field_name _name = get_field in
+          let result = List.fold_right
+              (fun  (pos, cc) code ->
                  begin match cc with
-                 | Tcoerce_primitive (id,p) -> 
+                 | Tcoerce_primitive (p, id) -> 
                      (if is_top rootpath then 
                         export_identifiers := id:: !export_identifiers);
-                     (transl_primitive Location.none p :: code, p.Primitive.prim_name ::name)
+                     transl_primitive Location.none p :: code
```

Comment:
previously we collect `p.prim_name`, now it is runtime_fields, this seems to be a subtle difference, do you observe any difference?

Reply:
This is the same as your question below on "tag info was giving the primitive's name, e.g. "%identity" instead of the field name.".